mention-bot.js

```diff
@@ -221,6 +227,26 @@ function getSortedOwners(
   return sorted_owners;
 }
 
+function getDefaultOwners(
+  files: Array<FileInfo>,
+  whitelist: Array<WhitelistUser>
+): Array<string> {
+  var owners = [];
+  var users = whitelist || [];
```

Comment:
whitelist is marked as non nullable in the type definition, so you don't need || []